src/Compilers/VisualBasic/Portable/Binding/Binder_Lookup.vb

```diff
@@ -899,8 +918,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     submission = submission.PreviousSubmission
                 Loop Until submission Is Nothing
 
-                If Not result.HasSymbol Then
-                    result.SetFrom(nonViable)
+                If result Is Nothing OrElse Not result.HasSymbol Then
+                    If result Is Nothing AndAlso nonViable.Kind <> LookupResultKind.Empty Then
```

Comment:
>nonViable.Kind <> LookupResultKind.Empty [](start = 49, length = 40)

It feels like this check might mess something, consider using ```!nonViable.IsClear``` instead

Reply:
The `result Is Nothing` isn't needed on the inner if, as it is covered by the outer if.